docs/general/administration/hardware-selection.md

```diff
@@ -9,55 +9,61 @@ This page covers what you need to know in order to select appropriate hardware f
 
 ## TL;DR
 
-For a Jellyfin server, the following is recommended:
+Below is a list of recommended specs to run Jellyfin. Even though Jellyfin will work on even lower end hardware, the following specs are recommended for a good experience.
 
-### Normal Server
+### Shared Items
 
-- CPU (With dGPU): Intel Core i5-2300, AMD Ryzen 3 1200 or better. (Geekbench 6 Multicore 1500 or better). *CPU vendor will not have a meaningful impact on quality and/or speed of transcoding.*
-- CPU (Without dGPU): Intel Pentium G4560, Intel Core i3-7100 or better. (Intel 7th gen or newer Pentium or better, excluding J and N series)
-- RAM: 8GB or more
-- Storage: 60GB SSD storage for Jellyfin files and transcoding cache.
-- Graphics: Intel HD 6xx (7th gen integrated graphics) or newer, Nvidia GTX 16 / RTX 20 series or newer (excluding GTX 1650). Intel is recommended over Nvidia. AMD and Apple Silicon are not recommended.
+Some component recommendations do not change regardless of the hardware configuration:
 
-:::note Intel "Atom" CPUs
+- Storage: 100GB SSD for Jellyfin files and transcoding cache. (Consider adding more on Windows 11)
+- Networking: Gigabit Ethernet Adapter or faster, WiFi or Powerline not recommended.
+- Internet Connection: At least 20mbps upload bandwidth (for remote access)
 
-Atom CPUs in this guide refer to any Intel Atom CPUs, as well as other low powered Intel CPUs such as J/M/N/Y series from other product lines. (eg. Pentium N5105, Pentium N6000, Core i3 N305). While these CPUs are branded as products from other product lines, they use the same microarchitecture as Intel's low powered chipsets and may be significantly less powerful and have less features than their names might otherwise suggest.
+### Hardware Encoder Quality
 
-:::
-
-:::note These are Recommended Specs
+Different vendors have different hardware encoder implementations and produce different results. Usually, newer generations within the same vendor will provide better results. The following is a quick comparison on the quality between vendors on modern products.
 
-These specs are the **recommended** specs to run Jellyfin. They are not minimum requirements, and it is certainly possible to run Jellyfin on lower end hardware.
+Apple ≥ Intel ≥ Nvidia >>> AMD
```

Comment:
I personally still think `≈` is more accurate than `≥`, but if you want to use `≥` it is still OK, but it would be better to add a comment that the result may vary due to encoder settings, and this currently reflects what Jellyfin produces

Reply:
![image](https://github.com/jellyfin/jellyfin.org/assets/12995396/b5319f89-fdf2-41f7-ac1d-351b6ccedb92)

I just see a table from an audio codec testing website, and this could suite our needs

Reply:
We can make a similar table to define what does these symbols mean. We don't have to use the exact symbol they are using, just make a clear definition could be good enough.

Reply:
I don't think ≥ is that confusing and I've seen it in many other places